packages/terra-paginator/src/ControlledProgressivePaginator.jsx

```diff
@@ -140,70 +143,67 @@ Previous
     return (
       <div className={cx(['paginator'])} role="navigation" aria-label="pagination">
         <div>
-          <a
+          <button
             aria-disabled={selectedPage === 1}
-            aria-label="first"
+            aria-label={intl.formatMessage({ id: 'Terra.paginator.first' })}
             className={cx(['nav-link', selectedPage === 1 ? 'is-disabled' : null])}
             tabIndex={selectedPage === 1 ? null : '0'}
             onClick={this.handlePageChange(1)}
             onKeyDown={this.handleOnKeyDown(1)}
+            type="button"
           >
-            First
-          </a>
-          <a
+            {intl.formatMessage({ id: 'Terra.paginator.first' })}
+          </button>
+          <button
             aria-disabled={selectedPage === 1}
-            aria-label="previous"
+            aria-label={intl.formatMessage({ id: 'Terra.paginator.previous' })}
             className={cx(['nav-link', 'previous', 'icon-only', selectedPage === 1 ? 'is-disabled' : null])}
             tabIndex={selectedPage === 1 ? null : '0'}
             onClick={this.handlePageChange(previousPageIndex)}
             onKeyDown={this.handleOnKeyDown(previousPageIndex)}
+            type="button"
           >
-            <span className={cx('visually-hidden')}>Previous</span>
+            <span className={cx('visually-hidden')}>{intl.formatMessage({ id: 'Terra.paginator.previous' })}</span>
```

Comment:
Could this be updated to use [Visually hidden text](https://engineering.cerner.com/terra-ui/#/components/terra-visually-hidden-text/visually-hidden-text/visually-hidden-text)?

I see more of these scattered around but I'm not going to comment on all of them.

Reply:
Thanks @Blackop778 - will investigate and give it a try.

Reply:
You can add `terra-visually-hidden-text` as a dependency in the package.json file.

Then run `npm run clean:install` in the root directory of the project.

Then update this line to the following:

```jsx
<VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.paginator.previous' })} />
```

Be sure to import the component in the file too:
```jsx
import VisuallyHiddenText from 'terra-visually-hidden-text';
```

Reply:
Completed. Added here https://github.com/cerner/terra-core/pull/2485/commits/a9892bd3801d58101322ebb40a768b38cd276906